src/loaders/micro-agent.js

```diff
@@ -0,0 +1,64 @@
+// loader files
```

Comment:
this represents the micro frontend loader, where all features are lazy-loaded, and only "manual" features are included.